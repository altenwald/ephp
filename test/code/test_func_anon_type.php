<?php

$sample = function($enclose, $data = 10) {
    print $enclose . __FUNCTION__ . "$enclose\n";
};

$sample("|");
print gettype($sample) . "\n";

var_dump($sample);
