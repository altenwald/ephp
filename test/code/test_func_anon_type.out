|{closure}|
object
object(Closure)#1 (1) {
  ["parameter"]=>
  array(2) {
    ["$enclose"]=>
    string(10) "<required>"
    ["$data"]=>
    string(10) "<optional>"
  }
}
